src/devtools/client/inspector/markup/markup.ts

```diff
@@ -127,7 +156,8 @@ class MarkupView {
 
   get eventTooltip() {
     if (!this._eventTooltip) {
-      this._eventTooltip = new HTMLTooltip(this.toolbox.doc, {
+      this._eventTooltip = new HTMLTooltip({
```

Comment:
You accidentally removed the first argument here

Reply:
But `toolbox` doesn't have `doc` property. And in other cases where `HTMLTooltip` is being instantiated, it makes the same mistake of passing an undefined prop from some inspector's toolbox object. It's also very silly because the first argument is not being used at all in the `HTMLTooltip` constructor. Either way, good catch.

I'll just pass a global `Document` object fr now, but maybe removing the first argument for the constructor of `HTMLTooltip` would be an even better fix.